tests/ClientFactoryTest.php

```diff
@@ -102,7 +102,12 @@ public function executesMockCommandSuccessfully(): void
 
         $this->assertTrue($createSite instanceof Command);
 
-        $client->execute($createSite);
+        $result = $client->execute($createSite)->toArray();
```

Comment:
If the `Result` is array-like then do we need to convert it `toArray` in order to access `$result['response']`?

Reply:
Nope, we [don't](https://github.com/basekit/php-api-client/pull/5/commits/ae5a2c44671fbe80c83baae44236688383132f2c) :)